src/blog/exploring_aleo_record_model.md

```diff
@@ -0,0 +1,178 @@
+
+
+An Aleo record is serialized in the following format:
+
+| Parameter  |             Type             | Description                                                                                         |
+|------------|:----------------------------:|-----------------------------------------------------------------------------------------------------|
+|     owner    |            address           |                      The address public key of the owner of the program record                      |
+|    data    |    `modifiable`   | A data payload containing arbitrary application-dependent information                               |
+|    nonce   |             group            |                            The serial number nonce of the program record                            |
+
+
+An example record: 
+```bash
+{
+  owner: aleo13ssze66adjjkt795z9u5wpq8h6kn0y2657726h4h3e3wfnez4vqsm3008q.private,
+  amount: 100u64.private,
+  _nonce: 5861592911433819692697358191094794940442348980903696700646555355124091569429group.public
+}
+```
+
+<b>Owner</b>
+<br/>
+`aleo13ssze66adjjkt795z9u5wpq8h6kn0y2657726h4h3e3wfnez4vqsm3008q`
+
+The record owner is an account address, and specifies the party who is authorized to spend the record.
+
+
+<b>Data</b>
+<br/>
+`100u64`
+
+The record can encode arbitrary application information. The "amount" key is the data payload that the record carries. 
+
+
+<b>Nonce</b>
+<br/>
+`5861592911433819692697358191094794940442348980903696700646555355124091569429group`
+
+The serial number nonce is used to create a unique identifier for each record, and is computed via a PRF evaluation of the address secret key ask of the owner and the record's serial number.
+
+<!-- Use this if iframe is working -->
+<!-- For a practical demonstration of record, see:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JIgrKv_Q6Jo?si=RRLiTa9n4iYMG63z" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe> -->
+
+For a practical demonstration of a record in Aleo, see [here](https://youtu.be/JIgrKv_Q6Jo?feature=shared).
+
+
+### Updating State
+In the record model, applications update their state by consuming records containing the old state, and producing new records that contain the updated state. Records that have been used will be marked as spent and cannot be used again.
+
+
+<p align="center" width="500">
+<img src="./images/utxo.png" alt="UTXO diagram">
+</p>
+
+<p align="center" width="100%">
+Diagram taken from <a href="https://adapulse.io/the-extensive-guide-on-eutxo-utxo-and-the-accounts-based-model/"> here</a>
+</p>
+<br>
+
+The consumption and production of records is typically done in a transition function. A transaction in Aleo consists of 32 transitions, of which one is reserved for the transaction fee. 
```

Comment:
Can a transaction contain less than 32 transitions? I though it could have an arbitrary amount of transitions with a limit of 32 max.

Reply:
Yes, it should be up to 32 transitions.